galileo/src/render/wgpu/mod.rs

```diff
@@ -598,7 +599,10 @@ impl WgpuRenderer {
             }
         });
 
-        if let Err(err) = self.device.poll(wgpu::PollType::Wait) {
+        if let Err(err) = self.device.poll(wgpu::PollType::Wait {
+            submission_index: None,
+            timeout: Some(std::time::Duration::from_secs(60)),
```

Comment:
60sec. was the behavior of previous version of `wgpu` (see: https://github.com/gfx-rs/wgpu/blob/trunk/CHANGELOG.md#wgpupolltypewait-has-now-an-optional-timeout). Maybe you want to have `timeout: None` ?